plexapi/client.py

```diff
@@ -126,7 +128,7 @@ def _loadData(self, data):
 
     def _headers(self, **kwargs):
         """ Returns a dict of all default headers for Client requests. """
-        headers = BASE_HEADERS
+        headers = self._base_headers
         if self._token:
             headers['X-Plex-Token'] = self._token
         headers.update(kwargs)
```

Comment:
Here's something that the previous code did not consider:

```python
dict1 = dict2
dict1.update(kwargs)
```

this updates `dict2` as well, i.e any such call would have modified `BASE_HEADERS` which would interfere with other calls, even from different classes.

likely never to be discovered as they all fill `X-Plex-Token` there with the same value.

this can be easily fixed by calling `headers = resrt_base_headers()` instead of assignment. so which way to go? optimize for using self._base_headers from constructor, or fix for correctness and use `reset_base_headers()` call in `_headers()`?

Reply:
It’s just a bug. We use copy other places.

Don’t break anything and use camelCase

Reply:
sorry, camelCase for what?